templates/layouts/_questionnaire.html

```diff
@@ -39,4 +42,22 @@
     {% endif %}
   {% endblock %}
 
+  {{
+    onsTimeoutModal({
+      "showModalTimeInSeconds": 60,
+      "serverSessionExpiryEndpoint": "/session-expiry",
+      "sessionExpiresAt": "",
+      "redirectUrl": "/signed-out",
```

Comment:
We should use `url_for` to fetch this and avoid hardcoding.

Reply:
`url_for` now used for both elements, so no hardcoded endpoints, thanks!